guides/getting-started.md

```diff
@@ -149,13 +149,22 @@ Here is a example configuration where `@angular/material`, `@angular/cdk/platfor
 System.config({
   // Existing configuration options
   map: {
-    // ...
     '@angular/material': 'npm:@angular/material/bundles/material.umd.js',
     
     // CDK individual packages
-    '@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
+    '@angular/cdk': 'npm:@angular/cdk/bundles/cdk.umd.js',
     '@angular/cdk/a11y': 'npm:@angular/cdk/bundles/cdk-a11y.umd.js',
-    // ...
+    '@angular/cdk/bidi':'npm:/@angular/cdk/bundles/cdk-bidi.umd.js',
+    '@angular/cdk/coercion':'npm:/@angular/cdk/bundles/cdk-coercion.umd.js',
+    '@angular/cdk/collections':'npm:/@angular/cdk/bundles/cdk-collections.umd.js',
+    '@angular/cdk/keycodes':'npm:/@angular/cdk/bundles/cdk-keycodes.umd.js',
+    '@angular/cdk/observers':'npm:/@angular/cdk/bundles/cdk-observers.umd.js',
+    '@angular/cdk/overlay':'npm:/@angular/cdk/bundles/cdk-overlay.umd.js', 
+    '@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
+    '@angular/cdk/portal':'npm:/@angular/cdk/bundles/cdk-portal.umd.js',
+    '@angular/cdk/rxjs':'npm:/@angular/cdk/bundles/cdk-rxjs.umd.js',
+    '@angular/cdk/scrolling':'npm:/@angular/cdk/bundles/cdk-scrolling.umd.js',
+    '@angular/cdk/table':'npm:/@angular/cdk/bundles/cdk-table.umd.js'
```

Comment:
At first glance, it looks like the `stepper` is missing here.

Reply:
That's true, @shaunak1111 see here for the list of `cdk` bundles: https://github.com/angular/cdk-builds/tree/master/bundles

Reply:
@DevVersion @donroyco I have added the necessary cdk modules stepper and layout in the docs and raised a PR.

https://github.com/angular/material2/pull/7445